.github/workflows/e2e-sample-apps-embedding-sdk.yml

```diff
@@ -0,0 +1,312 @@
+              const currentBranchVersion = versionMatch ? versionMatch[1] : null;
+
+              const supportedReleasesCount = Number(${{env.SUPPORTED_RELEASES_COUNT}});
+
+              const shouldRunTestsForVersion = (currentLatestVersion - currentBranchVersion) <= (supportedReleasesCount - 1);
+
+              return {
+                exclude_sample_app_names: 'shoppy',
+                sample_app_branch_name: branchName,
+                should_run_tests: shouldRunTestsForVersion,
+              };
+            }
+
+            return {
+              sample_app_branch_name: '',
+              should_run_tests: false,
+            };
+
+  build:
+    needs: [files-changed, get-build-requirements, get-sample-app-compatibility-data]
+    if: |
+      !cancelled() &&
+      github.event.pull_request.draft == false &&
+      needs.files-changed.outputs.e2e_all == 'true' &&
+      needs.get-sample-app-compatibility-data.outputs.should_run_tests == 'true'
+    runs-on: ubuntu-22.04
+    timeout-minutes: 25
+    env:
+      MB_EDITION: ee
+      INTERACTIVE: false
+      # make sure that builds on release branches get licenses, because we use them for releases
+      SKIP_LICENSES: ${{ github.event_name == 'pull_request' }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Prepare front-end environment
+        uses: ./.github/actions/prepare-frontend
+        with:
+          node-version: "${{ needs.get-build-requirements.outputs.node_version }}"
+      - name: Prepare back-end environment
+        uses: ./.github/actions/prepare-backend
+        with:
+          m2-cache-key: uberjar
+          java-version: "${{ needs.get-build-requirements.outputs.java_version || 21 }}"
+
+      - name: Build uberjar with ./bin/build.sh
+        run: ./bin/build.sh
+
+      - name: Prepare uberjar artifact
+        uses: ./.github/actions/prepare-uberjar-artifact
+        with:
+          name: metabase-ee-${{ github.event.pull_request.head.sha || github.sha }}-uberjar
+
+  reset-e2e-test-comment:
+    if: ${{ github.event_name == 'pull_request' }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Remove test results comment
+        uses: actions/github-script@v7
+        with:
+          script: | # js
+            const { owner, repo } = context.repo;
+
+            const { data: comments } = await github.rest.issues.listComments({
+              owner,
+              repo,
+              issue_number: context.issue.number,
+            }).catch(console.error);
+
+            if (!comments) {
+              return;
+            }
+
+            const comment = comments.find((comment) => comment.body.includes('## sample-apps-embedding-sdk-e2e failed on '));
+
+            if (comment) {
+              await github.rest.issues.deleteComment({
+                owner,
+                repo,
+                comment_id: comment.id,
+              });
```

Comment:
Would this delete and create a new comment instead of updating existing ones? I'd rather update the existing comment, so we could see the past status if needed. What do you think?

Reply:
We have the same behavior for regular e2e tests.
Also an original comment may be lost. It's much easier to see that something is failing at the bottom of the PR, so you always know that it is a fresh failure